recipes/pgsqlite/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pgsqlite" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pgsqlite-{{ version }}.tar.gz
+  sha256: 9ca2fc41741b8570ae5052391af12ada6dc00f1f339c0deb0e0882aeffc90ac8
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - python >=3.9
+    - sqlite-utils >=3.28
+    - psycopg >=3.1
+    - psycopg-c >=3.1
+    - structlog >=22.1.0
+    - sqlglot >=6.1.0
+
+test:
+  source_files:
+    - pgsqlite
+  imports:
+    - pgsqlite
+  commands:
+    - python ./pgsqlite/pgsqlite.py -h  # [unix]
```

Comment:
Tests seem to pass on Windows, why not run them there?

Reply:
I can try that